docs/pywbemclicommands.rst

```diff
@@ -149,17 +160,16 @@ See :ref:`pywbemcli class enumerate --help` for details.
 Class find command
 ^^^^^^^^^^^^^^^^^^
 
-The ``class find`` command gets classes filtered by the CLASSNAME-GLOB argument (a
-Unix style pathname pattern expansion) in the target WBEM server across
-multiple namespaces. It displays the results as a simple list or a table
-of the namespaces and class names in each namespace.
+The ``class find`` command lists classes with a class name that matches the
+pattern specified in the ``CLASSNAME-GLOB`` argument in all namespaces of the
+connection, or in the specified namespaces if the ``-namespace``/``-n``
+command option is specified one or more times.
 
-If successful it displays a list of the namespaces and classnames. If the
-WBEM server returns unsupported or other errors, the command fails with an
-exception.
+The pattern is a Unix-style pathname pattern. No tilde expansion is done,
```

Comment:
We specifically defined GLOB here but it is also defined in the appendix.  I put it in the appendix with the idea that we may want to use it for more than one thing and we do use that :term: elsewhere.

Reply:
DONE - Renamed the term "GLOB" to "Unix-style path name pattern", and referenced it from class find.